docs/adr/000-template.md

```diff
@@ -0,0 +1,63 @@
+# 0. Template
+
+Authors:
+
+
+## Status
+
+&lt;_Draft, Proposed, Reviewing, Accepted, Rejected, Superseded_>
+
+
+## Problem Statement
+
+&lt;_Why does this ADR exist? What decision are you trying to reach, what problem are you trying to solve?_>
+
+
+## Goals
+
+* &lt;_Clearly list the goals which this work will try to achieve._>
+* &lt;_The goals should be achievable, not visionary statements._>
+
+
+## Non-goals
+
+* &lt;_If any of the goals might be interpreted too broadly, here's where you limit their scope._>
+
+
+## Current Architecture
+
+* &lt;_If something is already solving the problem (even partially), here's where you mention it._>
+
+
+## Proposed Architecture
+
+* &lt;_Present a high level overview of the proposed solution._>
+* &lt;_Make it concise and simple; put diagrams; be concrete, avoid using vague generalizations._>
+
+
+## Challenges
+
+* &lt;_Call out any challenges here, e.g. upskilling needed, specific resources needed, very aggressive timelines if known, etc._>
+
+
+## Alternatives Considered
+
+* &lt;_Were any other approaches to solving the issue considered? If yes, describe them here._>
+* &lt;_Focus on the differences in outcomes (both positive and negative) between the proposed architecture and the alternatives._>
+
+
+## Dependencies
+
+* &lt;_Does this proposal depend on other things also being in place? List them._>
+
+
+## Stakeholders
+
+* &lt;_Who needs to sign off on this proposal for it to be implementable?_>
+  * &lt;_Since these ADRs impact how the management services teams operates, the default set of stakeholders would consist of relevant team members (plus external stakeholders, if any)._>
+
+
+## Consequences
+
+* &lt;_What happens if this proposal is accepted and implemented?_>
+* &lt;_Start with immediate consequences, don't base your argument on what you think might happen 5 years from now._>
```

Comment:
Both stakeholders and consequences feels weird in our context which is: this github. But I am fine if you want them, it might make sense when other teams are involved.

I would really love to see some examples of c4 diagrams directly in the template (as links into `diagrams/`) so I don’t need to think and dig too much and I can immediately start hacking.